models/ActivityModel.ts

```diff
@@ -1,15 +1,11 @@
-import { Entity, BaseEntity, Column, Generated, PrimaryGeneratedColumn, Index, ManyToOne, JoinColumn } from 'typeorm';
+import { Entity, BaseEntity, Column, PrimaryGeneratedColumn, Index, ManyToOne, JoinColumn } from 'typeorm';
 import { pick } from 'underscore';
 import { ActivityType, PublicActivity, Uuid } from '../types';
 import { UserModel } from './UserModel';
 
 @Entity('Activities')
 @Index('sliding_leaderboard_index', ['timestamp', 'pointsEarned'], { where: '"pointsEarned" > 0' })
 export class ActivityModel extends BaseEntity {
-  @Column({ select: false })
-  @Generated('increment')
-  id: number;
-
```

Comment:
are we good to keep these since we're not using SQLite anymore?

Reply:
We don't use them since UUIDs are our primary keys now, so might as well.